### PR TITLE
Create tc39-outreach-2019-01-07.md

### LINKED ISSUE #14: Call to discuss TC39 issues?
Would anyone be interested in having an call some time to discuss new JavaScript language features under discussion at TC39 and how they would relate to Node.js? I'd be really interested in getting your input on whatever you have thoughts on, or answer any questions you have. The discussion could be in the style of the existing educator and framework calls, which you can learn more about at https://github.com/littledan/js-outreach-groups .

### PR DESCRIPTION
**Pull request by littledan**
Closes #14

### CODE REVIEW

File: minutes/tc39-outreach-2019-01-07.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+* MD: Experimenting would look different in different cases
+* BF: Here, there are existing date APIs, so that affects it. There's also the broad issue of, does this make it easier and harder to write programs, which we can do by examining existing ones. Node can make recommendations to champions about what to consider from Node Core, and leave it up to the champions for how to apply this advice.
+* MD: I think Joyee's issue about what's being requested is good, and then see if we can find people to volunteer to do that work.
+
+#### Built-in modules
+
+#### Top-level await
+
+#### Zones
+
```
Comment (benjamingr) [inline minutes/tc39-outreach-2019-01-07.md:92]: We are currently blocking this (on error handling)

Comment (benjamingr) [reply minutes/tc39-outreach-2019-01-07.md:92]: Probably for a good reason (after domains)

Comment (littledan) [reply minutes/tc39-outreach-2019-01-07.md:92]: What we decided in the meeting was that we should look into these topics in issues on this repository, and schedule meetings about the particular topic as necessary. Could you start an issue about Zones and describe the issue?

Comment (ljharb) [reply minutes/tc39-outreach-2019-01-07.md:92]: looks like https://github.com/domenic/zones/issues/9 has relevant info

Comment (littledan) [reply minutes/tc39-outreach-2019-01-07.md:92]: This thread is really long, and the last comment I see is from 2016. Let's start a new issue to explain the current status and pick out the relevant things from history.

Comment (ghermeto) [reply minutes/tc39-outreach-2019-01-07.md:92]: @littledan did you start a new issue?

**Pull request merged**